pygmt/mathops.py

```diff
@@ -0,0 +1,59 @@
+"""
+GMT modules for Mathematical operations on tables or grids
+"""
+from .clib import Session
+from .exceptions import GMTInvalidInput
+from .helpers import build_arg_string, fmt_docstring, kwargs_to_strings, use_alias
+
+
+@fmt_docstring
+@use_alias(C="cmap", T="series", G="truncate", H="output", I="reverse")
+@kwargs_to_strings(T="sequence", G="sequence")
+def makecpt(**kwargs):
+    """
+    Creates a static color palette table (CPT).
+
+    Full option list at :gmt-docs:`makecpt.html`
+
+    Parameters
+    ----------
+    cmap (C) : str
+        Selects the master color palette table (CPT) to use in the interpolation.
+        Full list of built-in color palette tables can be found at
+        :gmt-docs:`GMT_Docs.html#built-in-color-palette-tables-cpt`.
```

Comment:
This URL has moved to https://docs.generic-mapping-tools.org/latest/cookbook/cpts.html#built-in-color-palette-tables-cpt

Reply:
Ok, done.